src/pages/Auth/SignUpPage/signUpPage/SignUp.module.scss

```
@@ -0,0 +1,302 @@
+
+$active: #ff7f23;
+$passive: #c4c4c4;
+$input-background: #eeeeee;
+
+@mixin error-outline() {
+  border: 1px solid $active;
+}
+
+.template {
+  @include mixin.template;
+}
+
+.container {
+  width: 368px;
+  height: 800px;
+  margin-top: 208px;
+
+  @include media.media-breakpoint-down(mobile) {
+    width: 227px;
+    height: 500px;
+    margin: auto;
+  }
+}
+
+.form {
+  margin-top: 128px;
+  height: 500px;
+  margin-bottom: 128px;
+  position: relative;
+
+  @include media.media-breakpoint-down(mobile) {
+    width: 100%;
+    margin-top: 65px;
+    position: relative;
+    margin-bottom: 0;
+    height: 415px;
+  }
+
+  &__title {
+    @include mixin.font-title(40px);
+
+    @include media.media-breakpoint-down(mobile) {
+      display: none;
+    }
+  }
+
+  &__form {
+    margin-bottom: 24px;
+    position: relative;
+
+    @include media.media-breakpoint-down(mobile) {
+      width: 100%;
+      height: 57px;
+      margin-bottom: 16px;
+    }
+  }
+
+  &__blind-icon {
+    position: absolute;
+    right: 10px;
+    bottom: 5px;
+
+    @include media.media-breakpoint-down(mobile) {
+      position: absolute;
+      bottom: 6px;
+      right: 12px;
+      width: 16px;
+      height: 16px;
+    }
+  }
+
+  &__label {
+    font-weight: 500;
+    font-size: 14px;
+    padding: 5px;
+    width: 408px;
+
+    @include media.media-breakpoint-down(mobile) {
+      width: 100%;
+      font-weight: 500;
+      font-size: 14px;
+    }
+  }
+
+  &__error-icon {
+    display: none;
+
+    &--active {
+      display: inline-block;
+      padding: 0 10px;
+
+      @include media.media-breakpoint-down(mobile) {
+        display: none;
+      }
+    }
+
+    &--mobile {
+      width: 12px;
+      height: 11px;
+      margin-right: 12px;
+    }
+  }
+
+  &__input {
+    width: 356px;
+    height: 13px;
+    padding: 7px 0 8px 12px;
+    outline: none;
+    background: $input-background;
+    border-radius: 20px;
+    border: 1px solid $input-background;
+    font-size: 12px;
+    color: #222222;
+    position: relative;
+
+    @include media.media-breakpoint-down(mobile) {
+      width: 213px;
+      height: 13px;
+      padding: 8px 0 7px 11px;
+    }
+
+    // chrome 자동완성 background 제거
+    &:-webkit-autofill {
+      box-shadow: 0 0 0 1000px $input-background inset;
+    }
+
+    &::placeholder {
+      font-size: 12px;
+      color: #c4c4c4;
+    }
+
+    &--id {
+      width: 290px;
+
+      @include media.media-breakpoint-down(mobile) {
+        width: 215px;
+      }
+    }
+
+    &--email {
+      width: 156px;
+
+      @include media.media-breakpoint-down(mobile) {
+        width: 215px;
+      }
+    }
+
+    &--error {
+      @include error-outline;
+    }
+  }
+
+  &__error {
+    font-size: 11px;
+    color: $active;
+    display: inline-block;
+
+    @include media.media-breakpoint-down(mobile) {
+      display: none;
+    }
+
+    &--top {
+      color: $active;
+      font-size: 11px;
+    }
+  }
+
+  &__id-check-button {
```

디자인 상 PC랑 모바일이 다른데 색상이 있는 디자인이 더 이쁜 것 같아요!
그리고 해당 속성 추가해주시면 버튼 글자가 약간 위로 올라가는 걸 방지할 수 있습니다.
```suggestion
  &__id-check-button {
    line-height: 12px;
```

---

넵 수정하였습니다! 5c5727fc957b2d7717da58e6973e9e2cd9f15c08